src/modules/esl-carousel/renderers/esl-carousel.default.renderer.ts

```diff
@@ -99,7 +97,7 @@ export class ESLDefaultCarouselRenderer extends ESLCarouselRenderer {
     const index = normalize(this.currentIndex + (direction === 'next' ? 1 : -1), this.size);
 
     // Make sure there is a slide in required direction
-    this.reorder(index, direction !== 'prev');
+    this.reorder(direction === 'prev');
```

Comment:
Honestly, I don't see the benefits of type `ESLCarouselDirection = 'next' | 'prev'`.
I doubt that it has readability benefits.
It is just a boolean marker isNextDirection or isPrevDirection.
Using a boolean type can reduce code.